stonesoup/updater/recursive.py

```diff
@@ -0,0 +1,345 @@
+import copy
```

Comment:
I think the recursive updaters need adding to the documentation (docs/source/stonesoup.updater.rst)